core/config/config.go

```diff
@@ -430,6 +430,23 @@ func (c *Config) Init() error {
 	return c.parseChains()
 }
 
+// Return the schema to use for accessing the node.
+func (c *Config) Schema() (s string) {
```

Comment:
1. This name is too generic: when reading code it seems like this is a schema for config itself. Maybe `HttpSchema` (don't have a better name really)

2. this code base generally avoids named return params, which get confusing really quickly. Really no need to use in this case - it's just one if with two returns.